repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/rackspaceauthuser/inputstream/system-model.cfg.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+    <repose-cluster id="repose">
+
+        <nodes>
+            <node id="node1" hostname="localhost" http-port="${reposePort}"/>
+        </nodes>
+
+        <filters>
+            <filter name="${preceding-filter}"/>
```

Comment:
I saw this and thought you were going to do a super sexy unroll, now I has a sad.